### DISCUSSION TITLE
Estimated Cost of Building

### DISCUSSION THREAD

**Discussion by baaart** [vroland/epdiy#339] [Q&A] [answered]
I was wondering what is the estimated cost of building single driver v7?
Is it like 10-20$ or more like 50-100$ ? Any estimations from anyone?

**Comment by martinberlin** [comment, marked as answer]
Is hard to give an estimate since it depends on how many you will produce. But if you are going to make just a few, roughly the cost of parts alone is around 30 USD (very rough estimate)
Being the most expensive components: esp32S3, tps65185 and pca9535 IO expander. Sum to that also connectors that also cost depending on choice (check hardware repository)
Actually best possible estimate is to try sending 5 units to fabrication in JLCPCB, since you can do so till last step, and you are not obligated to confirm the order. Then you can check both components plus PCB manufacturing (around 10 to 15 USD for 5 / 10)
And apart of that depending if you will use assembly or not there is also additional costs, plus customs for the order that is around 20% at least here from Spain.

For me all this effort to make only one is too much. That’s why when I send to fabrication, I make at least 20 and the rest I sell in Tindie. As much you make, prices can go down, since materials cost less when getting them in larger quantities

**Comment by baaart** [comment]
Long term maybe i will need a few but initially just one for prototype, im also not god at soldering, so maybe its better to get one from Tindie then:) I will try to get in touch with you as i need to decide then which version is best for my needs.
I was just thinking i will learn soldering along the way but maybe not worth the time
